libbeat/outputs/redis/client.go

```diff
@@ -17,6 +18,9 @@ import (
 
 var (
 	versionRegex = regexp.MustCompile(`redis_version:(\d+).(\d+)`)
+
+	ackedEvents    = expvar.NewInt("libbeat.redis.published_and_acked_events")
```

Comment:
I suggest to name it 

```
libbeat.redis.published.events.acked
libbeat.redis. published.events.not_acked
```
This corresponds better with our metricbeat conventions.